### PR TITLE
Added alpine CI

### PR DESCRIPTION
**Pull request by adrian-kong**
Adding alpine distro with armv6, armv7, aarch64, s390x, or ppc64le architectures via run-on-arch composite action - #2141

### CODE REVIEW

**Comment by dbwiddis** [comment]
Thanks for the submission!  Looks like it's not working... perhaps this GHA isn't ready for prime time yet.

**Comment by dbwiddis** [comment]
These are all failing on not finding the JDK:
> Executing process failed: java.io.IOException: Cannot run program "/usr/lib/jvm/java-17-openjdk/bin/java" (in directory "/home/runner/work/oshi/oshi/oshi-core"): error=0, Failed to exec spawn helper: pid: 125, exit value: 1

I think the issue here is that the java executable available in the Ubuntu host is not accessible inside the QEMU vm.

Try adding java (`openjdk`?) to the `apk add maven musl` line.

**Comment by dbwiddis** [comment]
(looks like you had that in c6016298f27fd333af9e7541db1afe5b795ac2e5 but then backed it out?)

**Comment by adrian-kong** [comment]
Huh interestingly, some of them runs for hours whereas the others

```
  This command is not for general use and should only be run as the result of a call to
  ProcessBuilder.start() or Runtime.exec() in a java application
```

Maybe failing to run a command inside the docker?

**Comment by adrian-kong** [comment]
@dbwiddis 

Managed to get 5 failures on [aarch64](https://github.com/oshi/oshi/actions/runs/3564655261/jobs/5988880376), [ppc64le](https://github.com/oshi/oshi/actions/runs/3564655261/jobs/5988880651) !

**Comment by dbwiddis** [comment]
> Managed to get tests to run with 5 failures 

Woohoo!   I knew it was choking in the tests but now we have a stack trace!   It's the `ExecutingCommand.runNative()` that doesn't seem to be working.

**Comment by adrian-kong** [comment]
Wow drains 5+ hours of running armv7 and armv6

File: .github/workflows/alpine.yaml [hunk 1]
```diff
@@ -25,12 +25,11 @@ jobs:
...
-      - name: Test on Alpine
-        id: test-alpine
+      - name: Test on Ubuntu
         uses: uraimo/run-on-arch-action@v2
         with:
           arch: ${{ matrix.arch }}
-          distro: alpine_latest
+          distro: ubuntu22.04
           install: |
             apk update
```
Comment (mprins) [inline .github/workflows/alpine.yaml:34]: Ubuntu uses `apt`

Comment (adrian-kong) [reply .github/workflows/alpine.yaml]: oops

**Pull request closed without merging**